templates/elements/quote/quote.html.json

```diff
@@ -0,0 +1,7 @@
+{
+  "modifier_class": "decanter-width-one-half",
```

Comment:
@josephgknox, The one half width is provided here in the example data an is not a requirement.

Reply:
Right, agreed. The example (because it's one quote by itself) won't be impacted too much without it. For consistency, I'm wondering if not including grid wrappers when not necessary will ensure that our sample markup in the library doesn't include unnecessary stuff. I'm cool either way, though.